physlr/physlr.py

```diff
@@ -1300,6 +1303,8 @@ def determine_molecules_process(u):
 
     def physlr_molecules(self):
         "Separate barcodes into molecules."
+        if self.args.strategy not in [1, 2, 3, 4]:
```

Comment:
The list `[1,2,3,4]` is duplicated. Use `self.args.strategy not in strategy_switcher`